se_feedback/sprint-5/feedback_sprint_5_all.md

```diff
@@ -0,0 +1,97 @@
+|      |            |
+|------|------------|
+|Group | Rubber Duck Debuggers |
+|Assignment|Sprint 5|
+|Date|29/05/16|
+|TA|Bastiaan Reijm|
+
+#Sprint Feedback
+Feedback and Grades for Sprint 5.
+
+Total: **9.55**
+
+| User Story | Score |
+|------------|-------|
+| definition |  9   |
+| splitting  |  10     |
+| responsibility | 10  |
+
+| Learning from History | Score |
+|-----------------------|-------|
+| estimation            | 10    |
+| prioritisation        | 10    |
+| reflection            | 9   |
+
+## Notes
+* Well done
+* That Matthias took over Robin's task should be added in a note, it looks like both were assigned responsibility and then one member was removed
+* Why is the row Octopeer from Logging Data empty and what is it?
+* Blank page 3
+* Make sure everyone catches up again on their hours in the next sprints.
```

Comment:
I'll take option 1, I saw the footnote it didn't quite click. So you can disregard the comment. Thanks for pointing that out.